config/locales/main/en-EU.yml

```diff
@@ -482,6 +482,10 @@ en-EU:
         you_can_choose_vote_dates: "This proposal has been waiting for %{count} days for a voting date, so you are now allowed to choose one."
         no_contributes_yet: "No contributions here yet"
         start_votation_proposal_confirmation: "Are you sure you want to start immediatly the votation?"
+        debate_begin: "Begin of debate"
+        debate_end: "End of debate"
+        vote_begin: "Begin of voting"
+        vote_end: "End of voting"
```

Comment:
Yup, I've just found 
vote_starts_at: "Vote starts at"
vote_ends_at: "Vote ends at"
in activerecord locales. But can't find something similar for debate phase - so added it
debate_starts_at: "Debate starts at"
debate_ends_at: "Debate ends at"